### ISSUE TITLE
Need general discussion on HC term 'dataset publisher'

### ISSUE THREAD

**Issue by pzermoglio** [tdwg/hc#5]
The term `dataset publisher` is within General Information category of terms in HC. 
Current HC definition: 

> ‘The entity making the dataset available.’

It has been suggested that this term be mapped to `dwc:institutionCode`: 

> The name (or acronym) in use by the institution having custody of the object(s) or information referred to in the record.

There seem to some issues with this term:

- The current HC definition is ambiguous. “available” to whom? And how? Note that in the examples table provided in the HC publication there is no example for this term.
- If mapped/turned into `dwc:institutionCode`, it also becomes tricky, specially depending on the type of inventory (summary vs others) and the original sources used to populate the record (as for the part of the definition “information referred to in the record”). 
o	Note here about how ambiguous this is also for any kind of data shared, as a single record may contain information from myriad sources (e.g., references/sources for identifications, geographic determinations, etc., which are also “referred to” in the record).

- If the intention with this term is to point to the final publisher of the dataset, in IPT it would be the organization in the GBIF registry, which record has to exist a priori and which is not part of the EML document. Yet it could be the eml:dataset/creator/organizationName by convention. In any case, it would be part of an EML profile, and not of the Event Core nor the HC extension.
- If the intention is to point to the original sources (e.g., in a dataset shared through Map of Life, the institution(s) from which the original data contributing to that dataset were grabbed), `dwc:institutionCode` may be closer, yet definitely different from what the current HC definition may imply, and with the caveats mentioned above.

Some considerations: 
-	The term would not have to be used in every case (just as any other term wouldn’t).
-	There may be types of inventories for which it makes sense to use it and some not.
-	There has also been a suggestion of considering `dwc:ownerInstitutionCode`: 

> The name (or acronym) in use by the institution having ownership of the object(s) or information referred to in the record. 

o	Same issues, or maybe worse to define “ownership” here. 
o	Note that other terms deal with license and rights.

Long story short: messy, and probably worth a larger discussion. Thoughts welcome!

**Comment by tucotuco** [comment]
It seems important to define what is meant by publishing here as well as the intended purpose of this term. In addition to the observations in the previous comment, there may be spillover into bibliographic citations as well. Failing any insights on the vagaries, I think this is a good candidate to deprecate.
